packages/import-export/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM debian:stable-slim
+
+ARG POSTGRESQL_SERVER
+ENV POSTGRESQL_SERVER=${POSTGRESQL_SERVER}
+
+ARG BRANCH_NAME
+ENV BRANCH_NAME=${BRANCH_NAME}
+
+RUN apt-get update && apt-get install -y \
+    curl \
+    git \
+    postgresql-client \
+    python3 \
+    python3-pip
+
+# Install nodejs
+RUN curl -sL https://deb.nodesource.com/setup_13.x | bash - && apt-get install -y nodejs
+# Install azure cli
+RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
+# Install pipenv
+RUN pip3 install pipenv
```

Comment:
Group RUNs to reduce and remove cache to reduce memory use 
see https://github.com/SocialGouv/docker/blob/v0.29.0/infra-ansible-ci/Dockerfile#L4-L23